src/Alert/IntentAlert.js

```diff
@@ -1,28 +0,0 @@
-import PropTypes from 'prop-types';
-
-import INTENTS from './intents';
-
-const IntentAlert = ({ intent, children, Component, ...props }) => (
-  <Component
-    borderLeft={intent ? '3px solid' : null}
-    borderColor={`intent.${intent}.default`}
-    {...props}
-  >
-    {children}
-  </Component>
-);
```

Comment:
`IntentAlert` has been removed entirely.  These border styles moved to the helper.